src/SFML/Graphics/Transform.cpp

```diff
@@ -164,6 +164,9 @@ sfBool sfTransform_equal(sfTransform* left, sfTransform* right)
     CSFML_CHECK_RETURN(left, false);
     CSFML_CHECK_RETURN(right, false);
 
-    return (left->matrix == right->matrix) ? sfTrue : sfFalse;
+    for (int i = 0; i < 9; ++i) {
+        if (left->matrix[i] != left->matrix[i]) return sfFalse;
```

Comment:
This is just comparing the left vector to itself

Reply:
Woops, looks like a copy paste issue, should be `right->` on the right side of course!

Reply:
I'm fixing this in an upcoming PR

Reply:
Oops 😅